image_view/include/image_view/image_saver_node.hpp

```diff
@@ -68,6 +68,7 @@ class ImageSaverNode
 
 private:
   std::string g_format;
+  int fps_;
```

Comment:
We can leave `fps_` as an int since there's no weird math similar to what was happening in `secs_per_frame`, right?